### DISCUSSION TITLE
Material not loading?

### DISCUSSION THREAD

**Discussion by JPhilipp** [atteneder/glTFast#488] [Q&A] [answered]
Hi, and thanks for making available this cool library!

My issue is that after loading any of the files, no material is being associated. The 3D files are *.glb originally from Google Poly, and they look fine when opened in e.g. the default Windows displayer. For instance, the following should be a chessboard, but all of the two nodes have the material set as None:

![image](https://user-images.githubusercontent.com/1754503/192725411-e56ba407-4fbc-483c-a0c7-2a4e57e8a28e.png)

![image](https://user-images.githubusercontent.com/1754503/192726738-d187fa18-eca1-405a-9c92-ba3ffeca7115.png)

I'm using this loading method, but also tried other ways:

        string url = "file:///E:/AIWorldGenerator/Assets/Resources/Models/Glb/00f9MZIwA1V.glb";
        gltfAsset = gameObject.AddComponent<GLTFast.GltfAsset>();
        gltfAsset.url = url;

I've attached the sample as zip file, in my project it's just a glb file though. Thanks for any help!

[sample-chessboard.zip](https://github.com/atteneder/glTFast/files/9662823/sample-chessboard.zip)

**Comment by atteneder** [comment, marked as answer]
Hi,

This looks like a bug, so you should raise a bug report issue and not a discussion.

Opening a bug report issue would guide you in providing the required information to fix it:

```
**Describe the bug**
A clear and concise description of what the bug is.

**Files**

Attach or link to *.gltf/*.glb files that trigger the bug. 

In addition, make sure to run those files through the [glTF Validator](https://github.khronos.org/glTF-Validator) first. If you encounter errors or warnings, try to make sure they are not responsible for the issue and file a bug report with the software that generated the glTF file as well.

> Note: You have to ZIP archive them first in order for GitHub to accept the upload.

If your files are confidential:

- Try to create a similar, but intellectual-property-free glTF that reproduces the bug in the same way (so any community member can have a look)
- Otherwise, still create this issue and send the files (or a link to them) discretely via email

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - glTFast version
 - Unity Editor version [e.g. 2021.2.1f1]
 - Render Pipeline and version [e.g. Universal Render Pipeline 12.0]
 - Platform: [e.g. Editor, Windows Player, iOS]
 
additionally (if significant for the bug):

 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - For WebGL: Browser [e.g. stock browser, safari]

**Additional context**
Add any other context about the problem here.
```

P.S.: It's working here (2019 LTS built-in)

<img width="784" alt="image" src="https://user-images.githubusercontent.com/2593014/192774080-6baebee1-86fc-4908-96dd-ba33dca057f8.png">

**Reply by JPhilipp** [reply]
Thanks!

**Reply by JPhilipp** [reply]
The Issue is now [added](https://github.com/atteneder/glTFast/issues/489).
